client/app/scripts/liveblog-edit/directives.js

```diff
@@ -50,40 +50,116 @@ define([
                 }
             };
         })
-        .directive('draftPosts', ['api', 'postsService',
-            function DraftPostsDirective(api, postsService) {
-                DraftPostsController.$inject = ['$scope'];
-                function DraftPostsController($scope) {
-                    var mv = this;
-                    mv.posts = [];
-                    mv.postsInfo = {};
-                    mv.selectDraftPost = $scope.onDraftPostSeleted;
-                    function updateList() {
-                        postsService.fetchDrafts(
-                            $scope.blog._id,
-                            mv.posts,
-                            mv.postsInfo
-                        );
+        .directive('lbPostsList', [
+            'postsService', 'notify', '$q',
+            function(postsService, notify, $q) {
+
+                LbPostsListCtrl.$inject = ['$scope'];
+                function LbPostsListCtrl($scope) {
+                    var vm = this;
+
+                    function fetchPage() {
+                        // stop if the limit is reached
+                        if (angular.isDefined(vm.postsMeta.total) && vm.postsMeta.total <= vm.posts.length) {
+                            return false;
+                        }
+                        // Find the next page containing new posts
+                        var page = 1;
+                        while (vm.pagination.limit * page <= vm.posts.length) {
```

Comment:
updates mechanism is ok it is the old one, but the pagination you refactored it without being the purpose of this task or benign mention every where that it has changed. 
here is the example:
page one and two from a three page is shown, you receive 7 updates. page 3 is no composed of     7 posts from  page 2 posts and 7 from page 3.